lib/tasks/ci/licenses.rake

```diff
@@ -14,7 +14,7 @@ namespace :ci do
     desc 'Check license compliance of dependencies'
     task :compliance do
       STDOUT.puts 'Checking license compliance'
-      unless system("BUNDLE_GEMFILE='Gemfile' #{threescale_license_finder_path}")
+      unless system("BUNDLE_GEMFILE='#{gemfile}' license_finder")
```

Comment:
I'm more tempted to remove the BUNDLE_GEMFILE, so it can be generated for any BUNDLE_GEMFILE just passed from the environment.